### ISSUE TITLE
Update accessibility document

### ISSUE THREAD

**Issue by SiniCode** [UniversityOfHelsinkiCS/palaute#1586]
The current accessibility document was created in 2021. Time for update...

**Comment by SiniCode** [comment]
Already fixed:
- [x] User menu is not keyboard accessible
- [x] Main navigation that becomes a hamburger with large zoom is not navigable with keyboard. 
- [x] Linked images do not tell where the links take
- [x] External links do not convey the information about external redirect for screen readers
- [x]  Navigation bar is not identified as such for screen reader
- [x] Language selection is not clear with screen reader
- [x] Skip links are not available
- [x] Question type selector not keyboard accessible
- [x] Feedback results not accessible at all
- [x] Contrast issues, especially in summary
